src/event.rs

```diff
@@ -23,6 +26,43 @@ use std::fmt::{self, Debug, Formatter};
 // FIXME - See https://maidsafe.atlassian.net/browse/MAID-2026 for info on removing this exclusion.
 #[allow(clippy::large_enum_variant)]
 pub enum Event {
+    // =========== Client Events - Routing won't handle these ===========
```

Comment:
Could this be called CleintEvent and remove all the Client refs inside?

Reply:
A separate event enum you mean ? If so I think that's fine.

Reply:
Yes that is what I mean. :+1: